geo/src/algorithm/coords_iter.rs

```diff
@@ -313,6 +411,31 @@ impl<'a, T: CoordinateType + 'a> CoordsIter<'a> for Geometry<T> {
             Geometry::Triangle(g) => g.coords_count(),
         }
     }
+
+    fn exterior_coords_iter(&'a self) -> Self::ExteriorIter {
+        match self {
```

Comment:
Sometimes I wonder if we should adopt https://docs.rs/enum_dispatch/0.3.4/enum_dispatch/